src/dbnode/encoding/m3tsz/m3tsz.go

```diff
@@ -29,28 +29,36 @@ const (
 	// DefaultIntOptimizationEnabled is the default switch for m3tsz int optimization
 	DefaultIntOptimizationEnabled = true
 
+	// OpcodeZeroSig indicates that there were zero significant digits.
+	OpcodeZeroSig = 0x0
+	// OpcodeNonZeroSig indicates that there were a non-zero number of significant digits.
+	OpcodeNonZeroSig = 0x1
+
+	// NumSigBits is the number of bits required to encode the maximum possible value
+	// of significant digits.
+	NumSigBits = 6
```

Comment:
It would be great if we could use `IntIterator` and `XORIterator` to avoid needing to export these.  Thoughts?